pkg/domain/infra/abi/play.go

```diff
@@ -15,6 +15,13 @@ import (
 	"github.com/containers/common/libimage"
 	"github.com/containers/common/pkg/config"
 	"github.com/containers/image/v5/types"
+	"github.com/ghodss/yaml"
```

Comment:
Why is this change here?

Reply:
I think that is a left over of my IDE. It has auto sorting of imports enabled. The sorting is configured for our internal projects.

Reply:
@hikhvar Could you please amend and remove this.